src/components/create-guild/BasicInfo/components/TwitterUrlInput.tsx

```diff
@@ -29,7 +29,7 @@ const TwitterUrlInput = (): JSX.Element => {
   }, [value])
 
   return (
-    <FormControl isInvalid={!!errors?.socialLinks?.TWITTER} isRequired>
+    <FormControl isInvalid={!!(errors?.socialLinks as any)?.TWITTER} isRequired>
```

Comment:
`react-hook-forms` got updated in the meantime, I think we're safe to remove the `any` casting here (and where we included it in this PR)